config/config_notwin_test.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 The Prometheus Authors
```

Comment:
Could we name it config_unix_test.go?

http://blog.ralch.com/tutorial/golang-conditional-compilation/ also says we don't need to mention build tags if the files are named right.